### ISSUE TITLE
Dialog won't close in IE11 because this.activeElement doesn't implement focus()

### ISSUE THREAD

**Issue by matthew798** [ghosh/Micromodal#260]
This is directly elated to #87 

In my case, however, `this.activeElement` isn't null, it's of type `SVGSVGElement` which, in IE11, doesn't implement `focus()`.

My recommendation is to check if the element supports focus like this:
```js
if (this.activeElement && this.activeElement.focus){
    this.activeElement.focus();
}
```

**Comment by matthew798** [comment]
I've created a PR to fix this #261

**Comment by ghosh** [comment]
Fixed in `0.4.3`
